routes/loans.js

```diff
@@ -1,295 +1,90 @@
-    loan_end_date: "",
-  },
-  {
-    id: "8",
-    book_id: "4",
-    status: "returned",
-    book: {
-      id: "1",
-      user_id: "1",
-      title: "The Name of This Book is Secret",
-      author: "",
-      isbn: "",
-      summary: "",
-      visibility: "public",
-    },
-    owner: {
-      id: "1",
-      name: "Bob",
-    },
-    requester: {
-      id: "2",
-      name: "Sally",
-    },
-    owner_contact: "bob@bob.bob",
-    requester_contact: "sally@sally.sally",
-    accept_date: "",
-    request_date: "",
-    loan_start_date: "",
-    loan_end_date: "",
-  },
-];
-
-const loanedByMe = [
-  {
-    id: "1",
-    book_id: "2",
-    status: "pending",
-    book: {
-      id: "1",
-      user_id: "1",
-      title: "Dummy's Guide to Telekinesis",
-      author: "",
-      isbn: "",
-      summary: "",
-      visibility: "public",
-    },
-    owner: {
-      id: "1",
-      name: "Bob",
-    },
-    requester: {
-      id: "2",
-      name: "Sally",
-    },
-    owner_contact: "bob@bob.bob",
-    requester_contact: "sally@sally.sally",
-    accept_date: "",
-    request_date: "",
-    loan_start_date: "",
-    loan_end_date: "",
-  },
-  {
-    id: "2",
-    book_id: "1",
-    status: "accepted",
-    book: {
-      id: "1",
-      user_id: "1",
-      title: "Third Class Superhero",
-      author: "",
-      isbn: "",
-      summary: "",
-      visibility: "public",
-    },
-    owner: {
-      id: "1",
-      name: "Bob",
-    },
-    requester: {
-      id: "2",
-      name: "Sally",
-    },
-    owner_contact: "bob@bob.bob",
-    requester_contact: "sally@sally.sally",
-    accept_date: "",
-    request_date: "",
-    loan_start_date: "",
-    loan_end_date: "",
-  },
-  {
-    id: "3",
-    book_id: "3",
-    status: "loaned",
-    book: {
-      id: "1",
-      user_id: "1",
-      title: "Genius at Play",
-      author: "",
-      isbn: "",
-      summary: "",
-      visibility: "public",
-    },
-    owner: {
-      id: "1",
-      name: "Bob",
-    },
-    requester: {
-      id: "2",
-      name: "Sally",
-    },
-    owner_contact: "bob@bob.bob",
-    requester_contact: "sally@sally.sally",
-    accept_date: "",
-    request_date: "",
-    loan_start_date: "",
-    loan_end_date: "",
-  },
-  {
-    id: "4",
-    book_id: "4",
-    status: "returned",
-    book: {
-      id: "1",
-      user_id: "1",
-      title: "Cryptonomicon",
-      author: "",
-      isbn: "",
-      summary: "",
-      visibility: "public",
-    },
-    owner: {
-      id: "1",
-      name: "Bob",
-    },
-    requester: {
-      id: "2",
-      name: "Sally",
-    },
-    owner_contact: "bob@bob.bob",
-    requester_contact: "sally@sally.sally",
-    accept_date: "",
-    request_date: "",
-    loan_start_date: "",
-    loan_end_date: "",
-  },
-];
-
 router
   .route("/")
   .get(function (req, res) {
-    res.statusCode = 200;
-    res.json([fakeLoan, fakeLoan, fakeLoan, fakeLoan, fakeLoan]);
+    let user = req.session.userId;
```

Comment:
This variable should be `userId` since it's not a whole user object. It's less confusing that way